docs/quick-start.md

```diff
@@ -0,0 +1,110 @@
+# DAG Factory
+
+**DAG Factory** is a library for [Apache Airflow®](https://airflow.apache.org) that allows you to construct DAGs declaratively via configuration files.
```

Comment:
Any type of configuration files? Or is it specific to YAML?

Reply:
I have rephrased this to: "DAG Factory is a Python library [Apache Airflow®](https://airflow.apache.org/) that simplifies DAG creation using declarative YAML configuration files." WDYT?